aes_kw/Cargo.toml

```
@@ -0,0 +1,26 @@
+[package]
+name = "aes_kw"
+version = "0.1.0" # Also update html_root_url in lib.rs when bumping this
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/RustCrypto/KWs/"
+repository = "https://github.com/RustCrypto/KWs/"
+description = "Nist 800-38F AES Key Wrapping Method (KW)"
+keywords = ["crypto", "AES-KW", "KW"]
+categories = ["cryptography", "no-std"]
+readme = "README.md"
+edition = "2018"
+
+[dependencies]
+aes = "^0.7"
+byteorder = "^1.4"
+generic-array = "^0.14"
+lazy_static = "1.2.0"
+
+[dev-dependencies]
+hex-literal = "0.2"
+hex = "^0.4"
```

`^` (SemVer) version matching is the default, so including it in version specifiers is redundant:

```suggestion
[dependencies]
aes = "0.7"
byteorder = "1.4"
generic-array = "0.14"
lazy_static = "1.2.0"

[dev-dependencies]
hex-literal = "0.2"
hex = "0.4"
```